requirements.txt

```diff
@@ -2,6 +2,7 @@ jupyter>=1.0.0,<2
 jupyter_contrib_nbextensions>=0.5.1,<1
 matplotlib>=2.2.2,<3
 numpy>=1.16.3,<2
+palettable==3.3.0
```

Comment:
```suggestion
palettable>=3.3.0,<4
```

Reply:
It's usually best to specify a range in `requirements.txt` because libraries often update.